torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -103,8 +103,7 @@ Message RequestCallbackImpl::processRpc(
       // TODO: make this asynchronous
       std::shared_ptr<OwnerRRef<IValue>> rref =
           ctx.getOrCreateOwnerRRef<IValue>(srf.rrefId());
-      return std::move(RRefFetchRet(RRefFetchRet({rref->getValue()})))
-          .toMessage();
+      return std::move(ScriptRRefFetchRet({rref->getValue()})).toMessage();
```

Comment:
Is std::move still needed on a temporary? IIRC a temporary qualifies as an rvalue.

edit: I checked and it should work without std::move (per godbolt): https://godbolt.org/z/-eeLnT

Reply:
Will address this on followup PRs. Thanks!